src/helpers/uuid.ts

```diff
@@ -11,6 +11,26 @@ export class UuidGenerator {
     this.isFastIdStrategy = isFast;
   }
 
+  smallUuid(): string {
```

Comment:
Maybe worth to add a comment to explain the difference between the two ? IMO the name is not very meaningful, but I do not have a better idea right now